File: gdscript/src/main/java/gdscript/psi/GdIfSt.java
package gdscript.psi;

import java.util.List;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public interface GdIfSt extends GdStmt {

  @NotNull
  List<GdElifSt> getElifStList();

  @Nullable
  GdElseSt getElseSt();

  @Nullable
  GdExpr getExpr();

  @Nullable
  GdStmtOrSuite getStmtOrSuite();

}

File: gdscript/src/main/java/gdscript/psi/GdIfSt.kt
package gdscript.psi

interface GdIfSt : GdStmt {
    val elifStList: List<GdElifSt>

    val elseSt: GdElseSt?

    val expr: GdExpr?

    val stmtOrSuite: GdStmtOrSuite?
}
